modules/v2x/common/v2x_proxy_gflags.cc

```diff
@@ -24,18 +24,55 @@
 namespace apollo {
 namespace v2x {
 
+//// IP & PORT
 DEFINE_string(grpc_client_host, "127.0.0.1", "grpc client host ip");
 DEFINE_string(grpc_server_host, "127.0.0.1", "grpc server host ip");
 DEFINE_string(grpc_client_port, "50100", "grpc client port num");
 DEFINE_string(grpc_server_port, "50101", "grpc server port num");
+DEFINE_string(grpc_debug_server_port, "50102", "grpc server debug port num");
+
+//// Channels
+DEFINE_string(localization_channel, "/apollo/localization/pose",
+              "localization channel name");
+DEFINE_string(perception_obstacle_channel, "/perception/obstacles",
+              "perception channel name");
+DEFINE_string(chassis_detail_channel, "/adu/carall", "chassis channel name");
+DEFINE_string(planning_channel, "/apollo/planning", "planning channel name");
+DEFINE_string(v2x_obstacle_channel, "/v2x/obstacles",
+              "v2x obstacles channel name");
+DEFINE_string(v2x_obu_trafficlight_channel, "/v2x/obu/internal/traffic_light",
+              "v2x obu trafficlight channel name");
+DEFINE_string(v2x_trafficlight_channel, "/v2x/traffic_light",
+              "v2x trafficlight channel name");
+DEFINE_string(v2x_trafficlight_for_hmi_channel, "/v2x/trafficlight_for_hmi",
+              "v2x trafficlight channel name");
+DEFINE_string(v2x_speedlimit_channel, "/v2x/v2x_speedlimit",
+              "v2x speedlimit channel name");
+DEFINE_string(v2x_rsi_channel, "/v2x/rsi", "v2x rsi channel name");
+
+//// Other Flags
 DEFINE_int64(x2v_trafficlight_timer_frequency, 10,
              "x2v trafficlight timer frequency");
-DEFINE_bool(debug_flag, false, "debug flag");
 DEFINE_int64(v2x_carstatus_timer_frequency, 10,
              "v2x carstatus timer frequency");
 DEFINE_string(hdmap_file_name,
               "/apollo/modules/map/data/sunnyvale_big_loop/base_map.bin",
               "hdmap file name");
-DEFINE_double(traffic_light_distance, 1000.0, "traffic light distance");
+DEFINE_double(traffic_light_distance, 250.0, "traffic light distance");
+DEFINE_double(heading_difference, 30.0 / 180.0, "max heading difference");
+DEFINE_int64(list_size, 6, "size of list which stores the traffic light data");
+DEFINE_int64(msg_timeout, 250, "timeout value which getting the msg from OBU");
+DEFINE_int64(sim_sending_num, 10, "the max sending times");
+DEFINE_bool(use_nearest_flag, true,
+            "use the hdmap interface get_forward_nearest_signals_on_lane flag");
+DEFINE_int64(spat_period, 150, "SPAT message periond");
+DEFINE_double(check_time, 0.5, "SPAT message periond");
+DEFINE_int64(rsu_whitelist_period, 3 * 1000, "get whitelist periond");  // 3s
+DEFINE_string(rsu_whitelist_name,
+              "/apollo/modules/v2x/conf/rsu_whitelist.txt",
+              "file name for RSU whitelist");
+DEFINE_int32(check_rsi_lane_num, 10, "");
+DEFINE_double(notifying_hmi_distance, 50.0, "");
+DEFINE_int32(find_all_lane_for_region, 10, "");
```

Comment:
Please provide the description.

Reply:
These flags are deprecated and now removed.